sql/index/pilosa/driver_test.go

```diff
@@ -44,7 +44,7 @@ func TestID(t *testing.T) {
 func TestLoadAll(t *testing.T) {
 	require := require.New(t)
 
-	path, err := ioutil.TempDir(os.TempDir(), "indexes")
+	path, err := ioutil.TempDir(os.TempDir(), "indexes-")
```

Comment:
should we set a path called `go-mysql-server` on temp dir to be able to delete it at the end of every test?

Reply:
The name is irrelevant to delete them at the end of every test. You just have to `os.RemoveAll(PathReturnedByTempDir)`

Reply:
yeah, we should at least make it consistent across all tests.

Reply:
Added `setup, cleanup` functions for tests' tmp dirs.